### PR TITLE
Make it possible to use .slather.yml on slather action

### PR DESCRIPTION
**Pull request by marcelofabri**
Fixes https://github.com/fastlane/fastlane/issues/4372

I'm not a big fan of the tests creating a file in the local dir, but haven't found a better way here.

### CODE REVIEW

File: fastlane/spec/actions_specs/slather_spec.rb [hunk 1]
```diff
@@ -62,14 +62,26 @@
...
-            slather({})
+            slather
           end").runner.execute(:test)
         end.to raise_error
       end
 
+      it "does not require project if .slather.yml is found" do
+        File.write('.slather.yml', '')
+
+        expect do
```
Comment (KrauseFx) [inline fastlane/spec/actions_specs/slather_spec.rb]: Why do we have this `expect` block?

**Comment by KrauseFx** [comment]
Looks great, thanks. I added one comment before it's ready 🚀

File: fastlane/spec/actions_specs/slather_spec.rb (hunk 1 above)
Comment (marcelofabri) [inline fastlane/spec/actions_specs/slather_spec.rb]: because I copied and pasted from somewhere else 😅 

I'l change that.

**Comment by marcelofabri** [comment]
@KrauseFx that expect block actually was hiding an exception 😱 

I fixed it. After your 👍, I'll squash again these commits.

**Comment by KrauseFx** [comment]
Looks great, thanks 👍

**Pull request merged**